README.md

```diff
@@ -919,13 +919,42 @@ extension your_class : RideRequestViewControllerDelegate {
 
 ## Example Apps
 
-Example apps can be found in the `examples` folder. To run build them, you need to use Carthage. (A quick overview of installing Carthage can be found in the **Getting Started** section.) From inside the `examples/Swift SDK` or `examples/Obj-C SDK` folder, run:
+Example apps can be found in the `examples` folder. To run build them, you can use Carthage or Cocoapods. 
+
+### Carthage
+(A quick overview of installing Carthage can be found in the **Getting Started** section.) From inside the `examples/Swift SDK` or `examples/Obj-C SDK` folder, run:
 
 ```
 carthage update --platform iOS
 ``` 
 This will build the required dependencies. Once you do that, open `Swift SDK.xcodeproj` or `Obj-C SDK.xcodeproj` in Xcode and run it.
 
+### CocoaPods
+You will have to remove Carthage dependencies. Remove Cartfile and Cartfile.resolved.
```

Comment:
I think this line could have more attention drawn to it, it also might be confusing which `Cartfile` & `Cartfile.resolved` to delete.

I think it might be better to have something like:

First, you will have to remove Carthage dependencies. 

Navigate to `examples/Swift SDK` or `examples/Obj-C SDK` and remove `Cartfile` and `Cartfile.resolved`. If you see a `Carthage` folder, remove that as well.